cincinnati/src/plugins/internal/graph_builder/openshift_secondary_metadata_parser/plugin.rs

```diff
@@ -295,6 +295,26 @@ impl OpenshiftSecondaryMetadataParserPlugin {
         }
     }
 
+    fn process_version(
```

Comment:
Lets declare this `pub` (same for `supported_version`) so that it could be reused in porting cincinnati-graph-data to Rust

Reply:
Wouldn't we just run this whole plugin in the one-shot CI tool?

Reply:
yeah, but it needs to be public so that we could import it in `cincinnati-graph-data` CI too

Reply:
@vrutkovs In the case which @wking describes we wouldn't need to use any code directly, but we'd run an appropriately configured graph-builder instead.

Reply:
so, eventually, in cincinnati-graph-data we'd run one-shot graph-builder? Sounds good to me, its a long shot though

Reply:
Is a one-shot invocation of this plugin more achievable?

Reply:
From the plugin perspective there's no other mode as one-shot as they don't compile to standalone binaries. It needs an embedding binary which in our case could be the graph-builder or the policy-engine. Just to mention it, I've toyed with the idea to make all of these the same binary with several sub-commands.